addons/packages/vsphere-cpi/1.22.5/bundle/config/vsphereconf.lib.txt

```diff
@@ -47,12 +47,12 @@ exclude-external-network-subnet-cidr = "(@=values.vsphereCPI.vmExcludeExternalNe
 router-path = "(@=values.vsphereCPI.nsxt.routes.routerPath @)"
 [NSXT]
 host = "(@=values.vsphereCPI.nsxt.host @)"
-(@ if values.vsphereCPI.nsxt.insecureFlag: -@)
+(@ if values.vsphereCPI.nsxt.insecure or str(values.vsphereCPI.nsxt.insecureFlag).lower() == "false" : -@)
 insecure-flag = true
 (@ else: -@)
 insecure-flag = false
 (@ end -@)
-(@ if values.vsphereCPI.nsxt.remoteAuth: -@)
+(@ if values.vsphereCPI.nsxt.remoteAuthEnabled or str(values.vsphereCPI.nsxt.remoteAuth).lower() == "false": -@)
```

Comment:
shouldn't this be `... or str(values.vsphereCPI.nsxt.remoteAuth).lower() == "true"`?

Reply:
Good catch

Reply:
this looks resolved now